src/StartMenu.ts

```diff
@@ -296,64 +298,82 @@ export class AEntryPointList implements IEntryPointList {
   private updateList(data: INamedSet[]) {
     const that = this;
 
+    this.$node.select('.header.subheader').text(`Predefined ${this.desc.description}`);
+    this.$node.append('ul').classed('namedSets', true);
+
+    const predefinedNamedSets = data.filter((d) => d.type !== ENamedSetType.NAMEDSET);
+    const customNamedSets = data.filter((d) => d.type === ENamedSetType.NAMEDSET);
+
+    if(this.$node.select('.customNamedSets').empty() && customNamedSets.length > 0) {
+      const customNamedSetsNode = this.$node.append('div').classed('customNamedSets', true);
+      customNamedSetsNode.append('div').classed('header subheader', true).text(`My ${this.desc.description}`);
+      customNamedSetsNode.append('ul');
+    } else if(customNamedSets.length === 0) {
+      this.$node.select('.customNamedSets').remove();
+    }
+
     // append the list items
-    const $ul = this.$node.select('ul');
-    const $options = $ul.selectAll('li').data(data);
-    const enter = $options.enter()
+    const $ul = this.$node.select('.namedSets');
+    const $customNamedSets = this.$node.select('.customNamedSets ul');
+
+    const $options = [$ul.selectAll('li').data(predefinedNamedSets), $customNamedSets.selectAll('li').data(customNamedSets)];
```

Comment:
you mean like
`
const predefinedList = this.$node.selectAll('.namedSets li').data(predefinedNamedSets);
const customList = this.$node.select('.customNamedSets ul li').data(...);
`